src/nav/new-relic-solutions.yml

```diff
@@ -182,12 +182,14 @@ pages:
             path: /docs/new-relic-solutions/build-nr-ui/custom-visualizations/build-visualization  
           - title: Add your visualization to a dashboard
             path: /docs/new-relic-solutions/build-nr-ui/custom-visualizations/add-to-dashboard-nerdgraph 
-      - title: Build your own New Relic apps
-        path: /docs/new-relic-solutions/new-relic-one/build-custom-new-relic-one-application
-      - title: Create a "Hello, World!" app in New Relic
-        path: /docs/new-relic-solutions/tutorials/build-hello-world-app
-      - title: Build a New Relic app with React hooks
-        path: /docs/new-relic-solutions/tutorials/build-react-hooks-app
+      - title: Build demo apps
+        pages:
+          - title: Build your own demo app
```

Comment:
This title is too different from the one on the page. Maybe, you could change one of them.
`Build your own demo app` <> `New Relic apps: build your own New Relic experiences`